light/data_model/db/environment.py

```diff
@@ -70,13 +79,17 @@ class DBNameKey(HasDBIDMixin):
     id and a name
     """
 
-    db_id: str = Column(String(ID_STRING_LENGTH), primary_key=True)  # type: ignore
-    name: str = Column(String(BASE_NAME_LENGTH_CAP), nullable=False, index=True, unique=True)  # type: ignore
-    status: DBStatus = Column(Enum(DBStatus), nullable=False, index=True)  # type: ignore
-    split: DBSplitType = Column(Enum(DBSplitType), nullable=False, index=True)  # type: ignore
+    db_id: Mapped[str] = mapped_column(String(ID_STRING_LENGTH), primary_key=True)
```

Comment:
Could this be related the SQLAlchemy issue I ran into today? It was complaining about the mapped columns too.

Reply:
Yup this was the cause of all of that.